### PR TITLE
Making My Branch equal to Main

### PR DESCRIPTION
**Pull request by JonathanTomaro**

### CODE REVIEW

**Comment by JonathanTomaro** [comment]
Not what I initially intended, but should be okay, just wanted to make mine set to what was in main, not merge both and make them equal in that way.

File: SoftwareDesignDocument..txt [hunk 1]
```diff
@@ -67,5 +67,19 @@ Code:
...
 Code:        
-	#define TANK_SHELL 64 in the item.h
+  #define TANK_SHELL 64 in the item.h
 	TANK_SHELL In items[] in the item.cpp file
+
+Req 1.3, 1.3.1, 1.3.3
+	When the user types "/play tank minigame" into the command, spawn the user into an arena at a fixed location.
+Code: 
+	//this function will get called upon "/play tank minigame" being inputed into the command line
+	void gameStart()
```
Comment (samschafer77) [inline SoftwareDesignDocument..txt:77]: This function looks good, the only thing missing is equipping both the plater tank and the enemy tank with the projectile blocks unless you have another way of implementing that.

**Review by samschafer77 — COMMENTED** [review]
The function needs to add the projectiles to the tanks as their active item.

**Comment by samschafer77** [comment]
Must add projectile to main hand of player when function is called.

**Pull request closed without merging**